test/src/com/fagi/guitests/SearchUserTests.java

```diff
@@ -241,11 +276,11 @@ public void WhenClientShowsSearchResults_ItShouldBeInTheOrderThatTheyWereReceive
         Label label4 = robot.from(contactNodes.get(3)).lookup("#userName").query();
         Label label5 = robot.from(contactNodes.get(4)).lookup("#userName").query();
 
-        Assertions.assertEquals(username1, label1.getText());
-        Assertions.assertEquals(username2, label2.getText());
-        Assertions.assertEquals(username3, label3.getText());
-        Assertions.assertEquals(username4, label4.getText());
-        Assertions.assertEquals(username5, label5.getText());
+        FxAssert.verifyThat(label1, LabeledMatchers.hasText(username1));
```

Comment:
Should this be an assertAll?

Reply:
Fixed https://github.com/miniwolf/Fagi/pull/18/commits/fd80b073521f6d57f05218bd1db2a9f30e943d31